sylvia/src/lib.rs

```diff
@@ -27,3 +27,5 @@ pub use {
     cosmwasm_schema as cw_schema, cosmwasm_std as cw_std, schemars, serde,
     serde_cw_value as serde_value, serde_json_wasm as serde_json,
 };
+#[cfg(feature = "orch")]
```

Comment:
This will make it visible in docs as reexported if `cw_orch` is enabled.

```suggestion
#[cfg_attr(docsrs, doc(cfg(feature = "cw_orch")))]
#[cfg(feature = "orch")]
```